docs/REVIEWING_COPILOT_PR.md

```diff
@@ -0,0 +1,116 @@
+# Reviewing GitHub Copilot Authored Pull Requests
+
+This guide explains how to review pull requests that have been authored by GitHub Copilot. GitHub Copilot is an AI pair programmer that can help you write code faster and with fewer errors. It can also be assigned to issues to implement solutions and submit pull requests for review.
+
+## Prerequisites
+
+Before you begin reviewing a Copilot-authored PR, ensure you have:
+
+- A GitHub account with permissions to review PRs in the repository
+- Basic familiarity with Git and GitHub PR workflow
+- Basic understanding of the Innovation Engine project structure
+
+## Step-by-Step Guide
+
+### 1. Opening the PR and Starting the Review
+
+1. Navigate to the Pull Requests tab in the repository
+2. Locate the PR created by GitHub Copilot (usually with "Copilot" as the author)
+3. Click on the PR title to open it
+4. Read through the PR description to understand what changes Copilot has made
+5. Review the "Files changed" tab to get an overview of which files were modified
+
+### 2. Using GitHub Codespaces for Review
+
+GitHub Codespaces provides a complete, configurable dev environment directly within GitHub, making it ideal for reviewing PRs:
+
+1. From the PR page, click on the "Code" dropdown button
+2. Select "Review in codespace" option
+3. Wait for the codespace to be created and initialized
+4. Once the codespace is ready, you'll have a full VS Code environment in your browser
+
+### 3. Setting Up the Environment
+
+To properly test the changes, you'll need to install the Azure CLI:
+
+```bash
+# Install Azure CLI
+curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+```
+
+Verify the installation:
+
+```bash
+az --version
+```
+
```

Comment:
If the user installed az cli they will need to login. Add instructions for doing so, bearing in mind that it will be necessary to use --use-device-code